src/peer/negotiator.js

```diff
@@ -477,6 +477,52 @@ class Negotiator extends EventEmitter {
   }
 
   /**
+   * Get map object describes which kinds of tracks should be marked as recvonly
+   * @param {Object} options - Options of peer.call()
+   * @return {Object} Map object which streamTrack will be recvonly or not
+   */
+  _getReceiveOnlyState(options) {
+    const state = {
+      audio: false,
+      video: false,
+    };
+
+    const stream = options.stream;
+    const hasStream = stream instanceof MediaStream;
```

Comment:
(This is my preference)
I think using `options.stream` directly is fine because the name `options.stream` sounds self-explanatory.

```js
const hasStream = options.stream instanceof MediaStream;
```

Reply:
Variable `stream` is used after this line too for simplicity. But it's ok, I'll fix them.

```js
const stream = options.stream;
const hasStream = stream instanceof MediaStream;
const hasAudioTrack = hasStream ? stream.getAudioTracks().length !== 0 : false;
const hasVideoTrack = hasStream ? stream.getVideoTracks().length !== 0 : false;
```

↓

```js
const hasStream = options.stream instanceof MediaStream;
const hasAudioTrack = hasStream ? options.stream.getAudioTracks().length !== 0 : false;
const hasVideoTrack = hasStream ? options.stream.getVideoTracks().length !== 0 : false;
```

Reply:
I see, this depends exactly on personal preference; I just prefer cutting assigning variable sometimes.